pkgs/development/libraries/gstreamer/devtools/default.nix

```diff
@@ -35,15 +35,24 @@ stdenv.mkDerivation (finalAttrs: {
     "dev"
   ];
 
-  src = fetchurl {
-    url = "https://gstreamer.freedesktop.org/src/gst-devtools/gst-devtools-${finalAttrs.version}.tar.xz";
-    hash = "sha256-7/M9fcKSuwdKJ4jqiHtigzmP/e+vpJ+30I7+ZlimVkg=";
+  src = fetchFromGitLab {
```

Comment:
Why this unrelated change?

Reply:
I thought it might be needed at the beginning of digging into this, though indeed it turned out not to be. I left it in as it seemed like an improvement anyway - but if you'd prefer me to move it to a separate commit or PR that's fine with me.